_meta/ecs-migration.yml

```diff
@@ -0,0 +1,154 @@
+# The ECS migration file contains the information about all the fields which are migrated to ECS in 7.0.
+# The goal of the file is to potentially have scripts on top of this information to convert visualisations and templates
+# based on this information in an automated way and to keep track of all changes which were applied.
+#
+# The format of the file is as following:
+#
+# - from: source-field-in-6.x
+#   to: target-filed-in-ECS
+#   # Alias field is useful for fields where there is a 1-1 mapping from old to new
+#   alias: true-if-alias-is-required-in-6x (default is true)
+#   # Copy to is useful for fields where multiple fields map to the same ECS field
+#   copy_to: true-if-field-should-be-copied-to-target-in-6x (default is false)
+
+- from: beat.hostname
+  to: agent.server.host.name
+
+- from: beat.name
+  to: agent.server.type
+
+- from: beat.version
+  to: agent.server.version
+
+- from: meta.cloud.availability_zone
+  to: cloud.availability_zone
+
+- from: meta.cloud.instance_id
+  to: cloud.instance.id
+
+- from: meta.cloud.instance_name
+  to: cloud.instance.name
+
+- from: meta.cloud.machine_type
+  to: cloud.machine.type
+
+- from: meta.cloud.project_id
+  to: cloud.project.id
+
+- from: meta.cloud.provider
+  to: cloud.provider
+
+- from: meta.cloud.region
+  to: cloud.region
+
+- from: context.service.agent.name
+  to: agent.name
+
+- from: context.service.agent.version
+  to: agent.version
+
+- from: context.system.architecture
+  to: host.architecture
+
+- from: context.system.ip
+  to: host.ip
+
+- from: context.system.platform
+  to: host.os.platform
+
+- from: context.request.method
+  to: http.method
```

Comment:
Spans do have a `context.http.method`, I think that should also be mapped to `http.method`.

Reply:
We can't alias this since it's not indexed and would be 1-many for users that don't split indices on `processor.event`.  I think since it's display only we could have this write to `http.method` in the next major and note that it won't show up for 6.x indices, including 6.6+.   Should we consider having the UI fallback from `http.method` to `context.request.method` for the medium/long term?

Reply:
`context.http.method` is not indexed, it is part of the mapping for non-indexed fields in https://github.com/graphaelli/apm-server/pull/1

Reply:
So I think we are good with just storing it at an ECS compliant position. Feel free to close this conversation if you agree.

Reply:
Agreed.  Started documenting non-indexed fields here: https://github.com/elastic/apm-server/pull/1609/commits/3ce04c0ae7b6f5a3b408a94fb187d253c3b4ae84, we can use that for fields we move to in 7.0 too.